pommerman/envs/v1.py

```diff
@@ -67,6 +67,8 @@ def collapse(r, c):
                     else:
                         new_bombs.append(b)
                 self._bombs = new_bombs
+            if utility.position_is_flames(board, (r, c)):
```

Comment:
Why do we actually need to do this check?

Reply:
Technically not needed - however it hopefully limits the number of times we need to execute the line below and as a side effect documents why we need it.